demo/Makefile

```
@@ -10,15 +10,23 @@ compose-integration:
 bump-deps:
 	./bump-deps.sh
 
-PHONY: plugin-build plugin-generate plugin-build-binary plugin-build-integration
+PHONY: plugin-build plugin-generate plugin-build-bun-binary plugin-build-go-binary plugin-build-integration
```

checkmake (0.2.2)

[warning] Target "PHONY" should be declared PHONY.

(phonydeclared)